app/_layouts/extension.html

```diff
@@ -216,13 +304,47 @@
 
 {{ plugin_config_for_route_admin_api }}
 
+`<route>` is the `id` or `name` of the Route that this plugin configuration
```

Comment:
I think per a better style, we should not start with a code element.

```suggestion
The `<route>` is the `id` or `name` of the Route that this plugin configuration
```

Reply:
This is referencing an item in the code sample and I do not think adding an article is appropriate here.